src/addrman.h

```diff
@@ -536,12 +536,12 @@ class CAddrMan
     }
 
     //! Mark an entry as accessible.
-    void Good(const CService &addr, bool test_before_evict = true, int64_t nTime = GetAdjustedTime())
+    void Good(const CService &addr, int64_t nTime = GetAdjustedTime())
         EXCLUSIVE_LOCKS_REQUIRED(!cs)
     {
         LOCK(cs);
         Check();
-        Good_(addr, test_before_evict, nTime);
+        Good_(addr, true, nTime);
```

Comment:
nit: Might be good to use named args where fundamental literal types are used for clarity

```suggestion
        Good_(addr, /* test_before_evict */ true, nTime);
```

Reply:
done!